README.md

```diff
@@ -32,33 +32,31 @@ By participating, you agree to this Code.
 argus has one function: interact with a database whether it is internal or external.
 To enable this, argus has two endpoints: 1) individual items, and 2) buckets containing items.
 
-### Create Individual Item - `store/{bucket}/{uuid}` endpoint
+### Create Individual Item - `store/{bucket}/{id}` endpoint
 This endpoint allows for clients to `PUT` an object into argus.  The placeholder variables in the path must contain:
 
 * _bucket_ - The name used to indicate the resource type of which the stored data represents.  A plural form of a noun word should be used for stylistic reasons.
-* _uuid_ - The unique ID within the name space of the containing bucket.  It is recommended this value is the resulting value of a SHA256 calculation, using the unique attributes of the object being represented (e.g. `SHA256(<identifier>)`).  This will be used by argus to determine uniqueness of objects being stored or updated.
+* _id_ - The unique ID within the name space of the containing bucket.  It is recommended this value is the resulting value of a SHA256 calculation, using the unique attributes of the object being represented (e.g. `SHA256(<common_name>)`).  This will be used by argus to determine uniqueness of objects being stored or updated.  argus will not accept any values for this attribute that is not a 64 character hex string.
```

Comment:
I think we should accept a wider range of characters. `[a-zA-Z0-9_=-]` to allow for base64url encoded values or ulid encoded values.  Restricting the special characters helps eliminate the potential for injection issues.

Reply:
If you use base64url encoding, then you're pretty much guaranteed that it won't be 64 characters long.  The idea is that we want clients to use SHA256.  If a client wants to deviate from that, then they'll need to make their own adjustments for something like base64url encoding.